static/js/services/mediastream.js

```diff
@@ -42,7 +42,7 @@ define([
 		var connectMarker = null;
 
 		// Create encryption key from server token and browser name.
-		var secureKey = sjcl.codec.base64.fromBits(sjcl.hash.sha256.hash(context.Cfg.Token + uaparser().browser.name));
+		var secureKey = sjcl.codec.base64.fromBits(sjcl.hash.sha256.hash(context.Cfg.Token + $.ua.browser.name));
```

Comment:
Please no additional jquery dependencies - it should be possible to use uaparser as AMD module without using jquery.